api/HotelsDatabaseContext.cs

```diff
@@ -36,6 +37,29 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     .IsUnicode(false)
                     .IsFixedLength(true);
             });
+
+            modelBuilder.Entity<Hotel>(entity =>
+            {
+                entity.Property(e => e.Address)
+                    .IsRequired()
+                    .HasMaxLength(155)
+                    .IsUnicode(false);
```

Comment:
Why `IsUnicode(false)`?

Reply:
Well, I have investigated this point, and on google maps places in China, for example, have bough china and english names, so I decided to use varchar instead of nvarchar
![image](https://user-images.githubusercontent.com/54571357/144068639-df74add7-658a-4d6c-b3df-9ae65e83c70a.png)

Reply:
What is best practice nowadays? Did you make a research? Pros/Cons for Unicode `true/false`?

Reply:
Well, I counted memory amount I need if use unicode, and accounts that have maximum fields length will cost 1 mb. Hotel text information about hotels will take quite a bit, if to compare with memory for pictures, but I could meet many problems with foreign users, so I changed encoding and now unicode is true for practically all fields in my api.